lib/galaxy/managers/tool_data.py

```diff
@@ -0,0 +1,81 @@
+
+from pathlib import Path
+from typing import Dict
+
+from galaxy import exceptions
+from galaxy.app import UniverseApplication
```

Comment:
Just merged https://github.com/galaxyproject/galaxy/pull/11142/files to provide a better way of doing this I think. Any chance you can rebase and import ``galaxy.app.structured_app.StructuredApp`` instead for stronger typing and better dependency management.

Reply:
Sure! I will do it. Let's see if this time I don't destroy the whole galaxy with my rebase attempt 😆

Reply:
Done!